tests/test_job.py

```diff
@@ -833,3 +833,48 @@ def test_job_shell_script_read_interpreter(hq_env: HqEnv):
     wait_for_job_state(hq_env, 1, "FINISHED")
 
     check_file_contents("out.txt", "Hello\n")
+
+
+def test_job_output(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+
+    hq_env.command(["submit", "--array", "1-2", "--", "bash", "-c", "echo 'hello'"])
+    wait_for_job_state(hq_env, 1, "FINISHED")
+    output = hq_env.command(["job", "cat", "1", "stdout"])
+    assert output.splitlines() == ["hello", "hello"]
+
+    output = hq_env.command(["job", "cat", "--tasks", "2", "1", "stdout"])
+    assert output.strip() == "hello"
+
+    output = hq_env.command(["job", "cat", "--tasks", "0", "1", "stdout"])
+    assert "Task 0 not found" in output.strip()
+
+    output = hq_env.command(["job", "cat", "1", "stderr"])
+    assert output == ""
+
+    hq_env.command(["submit", "--array", "1-2", "sleep"])
```

Comment:
Please write some string explicitly to stderr, either using Python (you can look at other tests how it's done) or Bash (https://stackoverflow.com/questions/2990414/echo-that-outputs-to-stderr).

On another system, sleep could output a different message and this test would fail.